judgels-frontends/raphael/src/modules/api/uriel/contest.ts

```diff
@@ -3,6 +3,7 @@ import { stringify } from 'query-string';
 import { APP_CONFIG } from 'conf';
 import { get, post } from 'modules/api/http';
 import { Page } from 'modules/api/pagination';
+import { ContestRole } from './contestWeb';
```

Comment:
Convention: blank line between absolute and relative imports

(Add blank line above this)